kornia/augmentation/container/augment.py

```diff
@@ -331,7 +337,12 @@ def forward(  # type: ignore[override]
         label: Optional[Tensor] = None,
         params: Optional[List[ParamItem]] = None,
         data_keys: Optional[List[Union[str, int, DataKey]]] = None,
-    ) -> Union[Tensor, List[Tensor], Tuple[Union[Tensor, List[Tensor]], Optional[Tensor]]]:
+    ) -> Union[
+        Tensor,
+        List[Tensor],
+        List[Union[Tensor, List[Tensor]]],
```

Comment:
Line 343 included L342 already.

Reply:
mypy disagree of this, because one is a simple list of tensors and the other can have lists and tensors (on the unpack of L299) -- maybe have other places with the wrong typing to this one be this way?

Reply:
Not sure. Anyway... as long as mypy is happy